### ISSUE TITLE
[nevil] Default font

### ISSUE THREAD

**Issue by vitkabele** [regolith-linux/regolith-look-extra#10]
Hello,
the nevil look sets default font to monospace font, which looks awful. Are you open to change it to something else?

**Comment by kgilmer** [comment]
Hi @vitkabele , absolutely improvements are welcome.  We'd need something that fits well with the monospace font (mononoki) that's already packaged for Ubuntu and Debian.  If you've got some ideas please create a PR :rocket:

**Comment by vitkabele** [comment]
Ok, I'll take a look into that

**Comment by sbmueller** [comment]
Why overwrite the font settings at all and not just leave it up to the gnome settings?

**Comment by kgilmer** [comment]
which font settings in particular @sbmueller ?  AFAIK there are these specified by settings, but users would like to customize bar font, bar icons, terminal, etc..

```
$ gsettings list-recursively | grep font-name
org.gnome.desktop.interface document-font-name 'Sans 12'
org.gnome.desktop.interface font-name 'Bitstream Vera Sans 12'
org.gnome.desktop.interface monospace-font-name 'BitstreamVeraSansMono Nerd Font 13'
```

**Comment by kgilmer** [comment]
The root-cause of this issue is that the nerdfonts that is specified for nevil does not have a non-monospace version.  I think it would be best to switch the font to another nerdfont that provides both to fix this issue.  I'll look into it.

**Comment by sbmueller** [comment]
Yes, I think that are the ones I mean (I set them on my machine via Gnome Tweaks).
After rebooting, my settings get overwritten again.
As a workaround, I have now commented out the font settings in the `nord` theme that I want to use. After that, everything is perfect 😃

**Comment by kgilmer** [comment]
> I think it would be more sensible that specific user settings take precedence over theme defaults.

Agreed.  The problem is how to _know_ that a given setting was explicitly set by a user with the intention of not wanting it to be overwritten.  Please let me know if you have any ideas here.

> I have now commented out the font settings in the nord theme that I want to use...

Glad that's working for you, and perhaps it would be nice if a flag could be set to prevent regolith-look from writing to dconf/gsettings.  LMK if you'd like to see such a feature in the future..

**Comment by kgilmer** [comment]
Also, just FYI, another way now that you could achieve the same result but be able to use different looks without modification is to update your copy of `/usr/share/regolith-look/default_loader.sh` and [simply disable the `gsettings set ...`](https://github.com/regolith-linux/regolith-look-default/blob/main/usr/share/regolith-look/default_loader.sh#L23) line that you would prefer left alone.  Your current approach is perfectly fine..
